sunbeam/hooks.py

```diff
@@ -40,6 +43,18 @@ def install(snap: Snap) -> None:
     LOG.debug(f'Copying {src} to {dst}...')
     shutil.copytree(src, dst)
 
+    # Tweak to use juju for zaza by creating symlink
+    # of local juju share folder within snap.
+    # Can be resolved by bug: https://bugs.launchpad.net/juju/+bug/1990797
+    env = SnapEnviron()
```

Comment:
Any particular reason why the [juju-client-observe-interface](https://snapcraft.io/docs/juju-client-observe-interface) won't work instead? I suspect its because the juju client will reference the $HOME/.local/share/juju and $HOME is remapped in snap context - so this doesn't feel entirely wrong, but I do agree with your comment that the bug would likely fix this as well (we shouldn't need to know where juju stores the credentials, etc).

Reply:
This is clear in later commits - and in fact does use the juju-client-observe interface.